bin/custom/questions/src/com/myapp/questions/populators/QuestionDataPopulator.java

```diff
@@ -0,0 +1,19 @@
+package com.myapp.questions.populators;
+
+import com.myapp.questions.data.QuestionData;
+import com.myapp.questions.model.QuestionModel;
+import de.hybris.platform.converters.Populator;
+import de.hybris.platform.servicelayer.dto.converter.ConversionException;
+
+import javax.annotation.Nonnull;
+
+public class QuestionDataPopulator implements Populator<QuestionModel, QuestionData> {
+    @Override
+    public void populate(@Nonnull QuestionModel questionModel, @Nonnull QuestionData questionData)
+            throws ConversionException {
+        questionData.setQuestionCustomer(questionModel.getQuestionCustomer().getName());
```

Comment:
if we populate a name to an attribute, it is better for attribute to display that by it's name. For example. QuestionData.questionCustomerName. Or initially it has to be a CustomerData instead, and here an additional populator needed to populate CustomerData from questionModel.getQuestiomCustomer

Reply:
better to name Model variables without a word "Model" in it - QuestionModel question